src/core/qgsproject.h

```diff
@@ -980,7 +980,7 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      *
      * \note Attached files are only supported by QGZ file based projects
      * \see collectAttachedFiles()
-     * \see attachedFile( fileName )
+     * \see attachedFile()
```

Comment:
@m-kuhn , your sip file is out of sync :)

Reply:
sip files are never out of sync, they synchronize exactly when they mean to.

![](https://media.giphy.com/media/AnTdtqwJSzW5W/giphy.gif)